cuda_pathfinder/docs/nv-versions.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "version": "latest",
+        "url": "https://nvidia.github.io/cuda-python/cuda-pathfinder/latest/"
+    },
+    {
+        "version": "1.1.0",
+        "url": "https://nvidia.github.io/cuda-python/cuda-pathfinder/1.1.0/"
+    },
+    {
+        "version": "1.0.0",
+        "url": "https://nvidia.github.io/cuda-python/cuda-pathfinder/1.0.0/"
+    }
```

Comment:
This should be removed because unless you rebuild the docs manually, the link won't be there and it'd be 404 error.

Reply:
Done (via `git revert`): commit ecc5c19c6bcffb0f59934cb2470271cfccdf5138

This for the explanation!